resources/views/partials/example-requests/python.blade.php

```diff
@@ -0,0 +1,27 @@
+```python
+import requests
+import json
+
+url = '{{ trim(config('app.docs_url') ?: config('app.url'), '/')}}/{{ ltrim($route['uri'], '/') }}'
+@if(count($route['bodyParameters']))
```

Comment:
As per [the docs](https://laravel-apidoc-generator.readthedocs.io/en/latest/generating-documentation.html#specifying-language-for-examples), you should be using `cleanBodyParameters` (odd name, I know). Also `boundUri` in the earlier line.